vignettes/SEIRD_Model_Compartments.Rmd

```diff
@@ -0,0 +1,261 @@
+The number of susceptible individuals evolves over time according to
+$$\frac{\text{d}S}{\text{d}t} = -S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) + \omega R,$$
+where $\beta_{I_X}$ and $\omega$ are positive parameter. The parameters $\beta_{I_X}$ denote the rate at which an infected individual of symptom level $X$ infects susceptible individuals,
+and $\omega$ denotes the rate at which recovered individuals become susceptible again.
+
+The number of exposed individuals evolves over time according to
+$$\frac{\text{d}E}{\text{d}t} = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s) - \kappa E,$$
+where $\kappa$ is a positive parameter denoting the rate at which exposed individuals become infectious. The incubation period thus lasts $1/\kappa$ days.
+The number of asymptomatic infectious individuals evolves over time according to
+$$\frac{dI_{a}}{dt} = \eta_{E->I_a} \kappa E - (\gamma_{I_a} + \mu_{I_a})I_{a},$$
+where $\eta_{E->I_a}$, $\gamma_{I_a}$ and $\mu_{I_a}$ are positive parameters. $\eta_{E->I_a}$ is the fraction of exposed individuals that, after the incubation period, became infectious but asymptomatic. $\gamma_{I_a}$ denotes the rate at which asymptomatic individuals recover from the disease and $\mu_{I_a}$ denotes the rate at which they die of the disease.
+
+The number of mildly asymptomatic infectious individuals evolves over time according to
+$$\frac{dI_{m}}{dt} = \eta_{E->I_m} \kappa E - (\gamma_{I_m} + \mu_{I_a})I_{m},$$
+where $\eta_{E->I_M}$, $\gamma_{I_M}$ and $\mu_{I_M}$ are positive parameters. $\eta_{E->I_a}$ is the fraction of exposed individuals that, after the incubation period, became infectious with mild symptoms. $\gamma_{I_M}$ denotes the rate at which mildly symptomatic individuals recover from the disease and $\mu_{I_M}$ denotes the rate at which they die of the disease.
+
+The number of severely asymptomatic infectious individuals evolves over time according to
+$$\frac{dI_{s}}{dt} = \eta_{E->I_s} \kappa E - (\gamma_{I_s} + \mu_{I_a})I_{s},$$
+where $\eta_{E->I_S}$, $\gamma_{I_S}$ and $\mu_{I_S}$ are positive parameters. $\eta_{E->I_a}$ is the fraction of exposed individuals that, after the incubation period, became infectious with severe symptoms. $\gamma_{I_S}$ denotes the rate at which severely symptomatic individuals recover from the disease and $\mu_{I_S}$ denotes the rate at which they die of the disease.
+
+The number of recovered individuals evolves over time according to
+$$\frac{\text{d}R}{\text{d}t} =  \mu_{I_a}  I_{a} + \mu_{I_m}  I_{m} + \mu_{I_s}  I_{s} - \omega R,$$
```

Comment:
Are the parameter names consistent between here and the documentation for the functions?